src/Microsoft.Health.Fhir.Core/Resources.resx

```diff
@@ -661,4 +661,11 @@
   <data name="SqlQueryProcessorRanOutOfInternalResourcesException" xml:space="preserve">
     <value>The query processor ran out of internal resources and could not produce a query plan. Please limit your parameters and retry.</value>
   </data>
-</root>
+  <data name="ResourceNotAllowedRestrictedByClinicalScopes" xml:space="preserve">
+    <value>Resource type {0} is not allowed, additional clinical scopes should be requested to allow searching using this type.</value>
+    <comment>{0} is a string of the name of the resource type.</comment>
+  </data>
+  <data name="ChainedResourceTypesNotAllowedDueToScope" xml:space="preserve">
+    <value>The (rev)chained resource type was not allowed.  Additional clinical scopes must be requested to allow searches using the chained type.</value>
```

Comment:
Nit: Change to `is` from `was`, to use the same pattern as the error message above?

Reply:
Updated.